pkg/async/notifications/factory_test.go

```diff
@@ -23,3 +31,13 @@ func TestGetEmailer(t *testing.T) {
 	// shouldn't reach here
 	t.Errorf("did not panic")
 }
+
+func TestNewNotificationsProcessor(t *testing.T) {
+	testSandboxProcessor := NewNotificationsProcessor(notificationsConfig, scope)
```

Comment:
Nit: could you call StartProcessing and StopProcessing here?

Reply:
No problem